packages/react-scripts/package.json

```diff
@@ -1,18 +1,18 @@
 {
```

Comment:
I don't recognize this packages/react-scripts, so I apologize if I'm asking a question that was already settled somewhere, but what is this? Is it a vendored copy of code from another repository? If so why are we vendoring it instead of installing it using yarn?

Reply:
Ignore this question, I was totally confused about which repository I was reviewing in.

Reply:
If you're curious, they're using lerna to manage multiple npm packages in 1 repo https://github.com/lerna/lerna. This `package.json` is the `package.json` specifically for the `react-scripts` npm package that the `create-react-app` npm package uses.

What's great about lerna is it lets you easily update all your dependencies across multiple npm packages. We should consider using it for managing our npm packages IMO.